db/openshift.deploy.yml

```diff
@@ -326,3 +329,86 @@ objects:
               dnsPolicy: "ClusterFirst"
               serviceAccountName: "${JOB_SERVICE_ACCOUNT}"
               serviceAccount: "${JOB_SERVICE_ACCOUNT}"
+  - kind: CronJob
+    apiVersion: "batch/v1"
```

Comment:
Maybe just "v1"?

Reply:
Hi Ian, what's the difference between batch/v1 and v1?